pitch/forms.py

```diff
@@ -0,0 +1,65 @@
+from django.forms import ModelForm
+from bootstrap_datepicker_plus.widgets import DateTimePickerInput
+from django.core.exceptions import ValidationError
+from pitch.models import Order
+from datetime import timedelta
+from django.utils import timezone
+from pitch.custom_fnc import convert_timedelta
+
+
+class RentalPitchModelForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        self.pitch = kwargs.pop("pitch")
+        super(RentalPitchModelForm, self).__init__(*args, **kwargs)
+
+    def clean_time_start(self):
+        data = self.cleaned_data["time_start"]
+        if data < timezone.now():
+            raise ValidationError("Chọn thời gian lớn hơn hôm nay.")
```

Comment:
Vẫn cho đặt trong ngày (sáng đặt cho tối đá) nên tránh gây hiểu nhầm cho người dùng e nha
```suggestion
            raise ValidationError("Thời gian bắt đầu phải lớn hơn giờ hiện tại")
```